src/ci/resources/papi_v2beta_application.conf

```diff
@@ -71,5 +71,14 @@ backend {
         include "papi_v2beta_provider_config.inc.conf"
       }
     }
+    Papiv2-Reference-Disk-Localization {
```

Comment:
what's this for?

Reply:
To be able to optionally enable this feature per configured backend. So here I added a backend, where it's enabled, for others it's implicitly disabled.